packages/taquito-michelson-encoder/src/tokens/token.ts

```diff
@@ -1,14 +1,15 @@
 import { MichelsonV1Expression, MichelsonV1ExpressionExtended } from '@taquito/rpc';
 import { TokenSchema } from '../schema/types';
+import { TaquitoError } from '@taquito/core';
 
 /**
  *  @category Error
  *  @description Error that indicates a failure when encoding invalid or incorrect data (e.g. if an address is expected but a number is received)
  */
-export abstract class TokenValidationError extends Error {
+export abstract class TokenValidationError extends TaquitoError {
   name = 'ValidationError';
```

Comment:
might want to correspond the name declaration with the class name

Reply:
Good catch, fixed